editor/src/core/webpack-loaders/loader-types.ts

```diff
@@ -0,0 +1,21 @@
+import { ProjectFile } from '../shared/project-file-types'
+
+export type MatchFile = (filename: string) => boolean
```

Comment:
Add a comment about how this can be extended in future to support https://webpack.js.org/configuration/module/#rule-conditions